asciidoc/client-applications.adoc

```diff
@@ -133,6 +133,64 @@ For details on how to configure multi-data center routing policies for a Causal
 ====
 
 
+=== Cluster bootstrap address resolution
+
+The 1.7 series drivers introduced the ability to intercept bootstrap address resolution.
```

Comment:
I am slightly confused here - The header says _Cluster bootstrap address resolution_ but the section seems to actually be about _intercepting_ bootstrap address resolution. Am I misunderstanding something or could the header be changed?

Reply:
@mariascharin The bootstrap address resolution applies only to routing driver, except python driver which invokes the configured callback function before dns resolution). However, we chose to document the common behaviour, anything to add @technige?